### PR TITLE
Transform according to orientation any tag; simplify scaling; catch a…

### PR DESCRIPTION
**Pull request by quite**
…ny exception

No longer scales up.

### CODE REVIEW

File: resize.py [hunk 1]
```diff
@@ -19,22 +20,24 @@ def resize(file, dir, key):
...
 
-        fn = f"{key}_{width}.jpg"
-        fp = os.path.join(dir, f"{key}_{width}.jpg")
-        img.save(fp, "JPEG")
-        generated_images.append((fn, fp))
+        img = ImageOps.exif_transpose(img)
 
+        for width in WIDTHS:
+            resized = img.copy()
+            resized.thumbnail((width, width))
```
Comment (rotsee) [inline resize.py:34]: Do we really want images to be squared? Don't we risk cropping them?

**Comment by quite** [comment]
I read the docs and it seems there won't be any cropping. It's just max
sizes.

Small images won't be scaled up though, so potentially "_1200" won't be that.

**Comment by rotsee** [comment]
Ok, great!

As long as there _is_ a _1200 version, it doesnt have to be scaled up. Those versions are used in the srcset attribute, so the effect for the end user is still the same (a small original being scaled up on some hi-res screens).

**Comment by quite** [comment]
Should we try it out? :)

**Comment by rotsee** [comment]
> Should we try it out? :)

Absolutely, I'll merge and deploy.
This also reminds me I should set up CI! Currently needs to be deployed by me...

**Comment by rotsee** [comment]
merged

**Pull request merged**